events/CloudFront/UpdateDistribution2020_05_31.json

```diff
@@ -9,6 +9,51 @@
     "mitreAttackTechniques": [
         "T1119 - Automated Collection"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1070 - Indicator Removal on Host",
+            "reason": "An attacker could modify CloudFront distribution settings to remove or alter logging configurations, thus deleting or hiding evidence of malicious activities."
+        },
+        {
+            "technique": "T1090 - Proxy",
+            "reason": "By updating CloudFront distribution, an attacker can route traffic through CloudFront, effectively hiding the origin of malicious traffic and obfuscating command and control communications."
+        },
+        {
+            "technique": "T1567 - Exfiltration Over Web Service",
+            "reason": "An attacker might reconfigure CloudFront to redirect sensitive data to an external endpoint under their control, facilitating data exfiltration over a web service."
+        },
+        {
+            "technique": "T1562 - Impair Defenses",
+            "reason": "Modifying CloudFront distribution settings can be used to impair security monitoring and defense mechanisms by disabling or altering configurations that are critical for security monitoring."
+        },
+        {
+            "technique": "T1560 - Archive Collected Data",
+            "reason": "An attacker might modify the CloudFront distribution to use cloud storage as a method to archive and exfiltrate collected data."
+        },
+        {
+            "technique": "T1497 - Virtualization/Sandbox Evasion",
+            "reason": "CloudFront configurations can be updated to delay or slow responses, making detection and analysis more difficult, effectively evading automated analysis systems."
```

Comment:
My point was that there could be malware deployed to a CloudFront distribution that activates only under certain conditions or delays its execution until it is almost too late, slowing down responses enough that it isnt too noticeable, but still measureable. Classic malware does this to evade antivirus detection since a long time.